src/content/app/help/state/helpSelectors.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * See the NOTICE file distributed with this work for additional information
+ * regarding copyright ownership.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { RootState } from 'src/store';
+
+export const getNavHistory = (state: RootState) => state.help.navHistory;
+
+export const getCurrentItem = (state: RootState) => {
```

Comment:
We should do something to the names of these selectors. "Item", as a name, is way too general. We should at least call it "history item"; or perhaps you could think of a better name.

Reply:
Renamed this to `currentHistoryItemIndex`

Reply:
✅

Reply:
The same should apply to getNextItem and getPreviousItem.

(And, by, the way, is there any reason to check that `currentHistoryItemIndex > -1`? Wouldn't `return navHistory[currentHistoryItemIndex] ?? null` always work?

Reply:
This is done as well ✅

Reply:
>(And, by, the way, is there any reason to check that currentHistoryItemIndex > -1? Wouldn't return navHistory[currentHistoryItemIndex] ?? null always work?

In that case it will always result in true even when it is -1:

Reply:
> In that case it will always result in true even when it is -1:

I am confused. I am not talking about returning `currentHistoryItemIndex ?? null`, but about returning `navHistory[currentHistoryItemIndex] ?? null`. `navHistory[-1]` will be undefnied.

Reply:
> > In that case it will always result in true even when it is -1:
> 
> I am confused. I am not talking about returning `currentHistoryItemIndex ?? null`, but about returning `navHistory[currentHistoryItemIndex] ?? null`. `navHistory[-1]` will be undefnied.

Ah yes that should work I think. Got carried away focussing too much on the -1 😁

Reply:
That's done now ✅